pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -25,6 +25,16 @@
     get_upcast_box,
 )
 
+
+def _get_dtype(obj):
```

Comment:
maybe better to pass a dtype and then call this assert_dtype ?

Reply:
changed